free/src/main/scala/cats/free/Yoneda.scala

```diff
@@ -35,7 +35,7 @@ object Yoneda {
   /**
    * `Yoneda[F, _]` is a functor for any `F`.
    */
-  implicit def catsFreeFunctorForYoneda[F[_]]: Functor[Yoneda[F, ?]] =
+  implicit def catsCofreeFunctorForYoneda[F[_]]: Functor[Yoneda[F, ?]] =
```

Comment:
I think the `Free` here is the package (like we have `catsDataFunctorForKleisli`.